src/ui/theme/components/button.ts

```diff
@@ -1,21 +1,86 @@
 const ButtonStyle = {
   // style object for base or default style
   baseStyle: {
-    borderRadius: 0,
+    borderRadius: 'none',
   },
   // styles for different sizes ("sm", "md", "lg")
   sizes: {},
   // styles for different visual variants ("outline", "solid")
   variants: {
-    outline: {
-      borderRadius: 0,
-    },
+    solid: variantSolid,
   },
-  // default values for `size` and `variant`
+  // default values for `size`, `variant`, `colorScheme`
   defaultProps: {
-    size: '',
-    variant: '',
+    size: 'md',
+    variant: 'solid',
+    colorScheme: 'light',
   },
 };
 
+/* Color Schemes:
+ ** Light, Sepia, Dark
+ * States:
+ ** Normal
+ ** Selected
+ ** Disabled
+ ** Hovered
+ */
+function variantSolid(props: any) {
+  const { colorScheme, isChecked } = props;
+  const common = {
+    _focus: {
+      'z-index': 1,
+    },
+  };
+
+  const toggleButtonBg = (uncheckedColor: string) =>
+    isChecked ? 'green.500' : uncheckedColor;
+  const toggleButtonColor = (uncheckedColor: string) =>
+    isChecked ? 'white' : uncheckedColor;
```

Comment:
One thing I wonder is whether the `Button` should have this toggle logic, or whether this should only be on a `ToggleButton`... We could put this logic to change the styles directly in the `ToggleButton` component as passing a `bg` or `color` prop will override the ones from the theme. Or we could add a `theme.components.ToggleButton` value with the styles for the toggle button. Not totally sure yet, but I don't think the "checked" logic needs to live in the normal button